ui/src/shared/components/Gauge.tsx

```diff
@@ -279,14 +281,23 @@ class Gauge extends Component<Props> {
     minValue,
     maxValue
   ) => {
-    const {prefix, suffix, decimalPlaces} = this.props
+    const {tickPrefix, tickSuffix, decimalPlaces} = this.props
+    let {prefix, suffix} = this.props 
     const {degree, lineCount, labelColor, labelFontSize} = GAUGE_SPECS
 
     const tickValues = [
       ..._.range(minValue, maxValue, Math.abs(maxValue - minValue) / lineCount),
       maxValue,
     ]
 
+    if(tickPrefix === "true"){
```

Comment:
despite being the default values, this seems a bit counterintuitive to have a string value set to `true`, is there any way we can have the defaults set to anything else? What happens if the user automatically sets the prefix or suffic to `"true"`?

Reply:
So the reason we are doing this as a string  instead of a boolean is in case in the future we will allow people to input their own custom ticks as a string. So right now we are setting it to string false or string true. In the UI it is a checkbox, so the user can not input anything themselves.

Reply:
Since we aren't currently focused on user input, I think it would be beneficial to keep things straightforward and have the types be a boolean option. Forward planning is always important, and you did a great job of investigating a solution for it, but since we don't know if this will be a feature request in the future, it might be a little strange for future developers to see `true` and `false` string values